docs/source/tags/taxonomy.md

```diff
@@ -20,6 +20,8 @@ Use with the following data types: audio, image, HTML, paragraphs, text, time se
 | [showFullPath] | <code>boolean</code> | <code>false</code> | Whether to show the full path of selected items |
 | [pathSeparator] | <code>string</code> | <code>&quot;/&quot;</code> | Separator to show in the full path |
 | [maxUsages] | <code>number</code> |  | Maximum number of times a choice can be selected per task |
+| [maxWidth] | <code>number</code> |  | Maximum width for dropdown |
```

Comment:
Replace dropdown with `drop-down` throughout this PR.

Reply:
why? that's widely used common term